Dockerfile

```diff
@@ -1,7 +1,18 @@
-FROM node:10
+FROM node:10 as builder
 WORKDIR /opt/app
 COPY package*.json ./
 RUN npm install
 COPY ./ ./
 RUN npm run build-dev
-CMD ["npm", "start"]
+
+FROM nginx:1.17.2
+WORKDIR /opt/app
+COPY --from=builder /opt/app/dist/wastexchange-fe ./wastexchange-fe
+COPY ./nginx.conf.template ./
+RUN apt-get update && apt-get install -y wget
```

Comment:
Should we also clean the downloaded packages that are not needed using:
```
sudo apt-get autoclean
```

Reply:
there are no partial packages to clean. so I think we can skip this

Reply:
not that we know of *right now* - but, imo, its a good practice to include for keeping the image size small. Please do incorporate.

Reply:
Done